_includes/sponsor-ad-banner.html

```diff
@@ -0,0 +1,3 @@
+<div class="row alert alert-success text-center mb-0 rounded-0" role="alert">
+    <h4>2023载歌在谷夏季歌手赛及中秋游园会火热招商中 <a href="/sponsorship/" class="alert-link">查看详情</a></h4>
```

Comment:
nit: The message body and the link are kinda close. Punctuation or full-width space might make it look easier.

```suggestion
    <h4>2023 「载歌在谷」夏季歌手赛、中秋游园会火热招商中！<a href="/sponsorship/" class="alert-link">查看详情</a></h4>
```

Smart usage of the `alert-success` class!

Reply:
use `b` instead, centered vertically issue is also auto resolved